templates/components/element_detail_tabs.html

```diff
@@ -0,0 +1,458 @@
+  }
+
+  .inline {
+    display: inline;
+  }
+
+  h3 {
+    color: #888;
+    #font-weight: bold;
+    margin-top: 20px;
+  }
+
+  #above-tab-content, #tab-content {
+    max-width: 950px;
+    margin: auto;
+  }
+
+  .control-text {
+    white-space: pre-wrap;
+    font-size: 11pt;
+    /*max-width: 700px;*/
+    font-family: trebuchet ms, sans-serif;
+    text-align: justify;
+    /*text-justify: inter-word;*/
+    line-height: 24px;
+  }
+
+  #control-lookup input {
+    width: 130px;
+  }
+
+  #combined_smt {
+    width: 80%;
+    /*margin: 12px 0 16px 0px;*/
+    padding: 8px;
+    /*display: none;*/
+  }
+
+  textarea {
+    overflow-y: scroll;
+    resize: vertical;
+  }
+
+  a {
+    color: #666;
+  }
+
+  #common-tab-count, #component-tab-count {
+    display: inline-block;
+    color: #666;
+    background-color: #dedede;
+    border-radius: 20px;
+    line-height: 1;
+    padding: 2px 6px;
+    font-size: 0.8em; font-weight: 600;
+  }
+
+  .panel-heading-smt, .panel-heading-smt-body {
+    white-space: pre-line;
+  }
+
+  .panel-heading a:hover {
+    text-decoration: none;
+    color: #000;
+  }
+
+</style>
+{% endblock %}
+
+{% block contextbar %}{% endblock %}
+
+{% block body %}
+<div style="margin-top: 30px">
+  <div class="container">
+
+  <div class="row" style="">
+    <div id="above-tab-content" class="row">
+      <!-- Page Title-->
+      <div style="float: left;">
+          <h2 class="control-heading" style="">
+            <a href="/controls/components" style="color: black;">Component Library</a> &gt; <a href="/systems/{{system.id}}/components/selected" style="color: black;">Component Detail</a>
+          </h2>
+      </div>
+
+    </div>
+  </div><!--/row-->
+
+  <div id="above-tab-content" class="row">
+    <div style="float: left; padding: 0 0 0 0em;">
+        <h2 class="control-heading" style="">
+            {{ element.name }}
+        </h2>
+    </div>
+  </div>
+
+<div>&nbsp;</div>
+
+  <div id="tab-content" class="row">
+    <!-- Nav tabs -->
+    <ul class="nav nav-tabs" role="tablist">
+      <li role="presentation" class="active"><a href="#component" aria-controls="component" role="tab" data-toggle="tab"><span class="glyphicon glyphicon-list-alt"></span> Component</a></li>
+      <li role="presentation"><a href="#component_controls" aria-controls="component_controls" role="tab" data-toggle="tab"><span class="glyphicon glyphicon-wrench"></span> Component Statements  &nbsp;<div id="common-tab-count">{{ impl_smts|length }}</div></a></li>
+      {% if enable_experimental_opencontrol %}
+        <li role="presentation"><a href="#oscal" aria-controls="oscal" role="tab" data-toggle="tab"><span class="glyphicon glyphicon-file"></span> OSCAL </a></li>
+      {% endif %}
+      {% if enable_experimental_opencontrol %}
+        <li role="presentation"><a href="#opencontrol" aria-controls="opencontrol" role="tab" data-toggle="tab"><span class="glyphicon glyphicon-file"></span> OpenControl </a></li>
```

Comment:
Small nit-pick about the use of spaces in list items. It looks like we're using them here to add space between them, but if there's way to add that space in styling, that may be preferred. Not a huge deal, though.

Reply:
@omarabed15 Yes, will leave the styling improving to others. Maybe work for January? Free feel to create an issue in Github.